resource/eastwood/config/clojure.clj

```diff
@@ -68,3 +68,11 @@
    :function-symbol 'clojure.core/eduction
    :arglists-for-linting '([& xform])
    :reason "eduction takes a sequence of transducer with a collection as the last item"})
+
+(disable-warning
+ {:linter :suspicious-test
+  :if-inside-macroexpansion-of #{'clojure.test/are}
+  ;; only omit the warning if :suspicious-test failed due to a `true` value:
+  :qualifier true
```

Comment:
The concept of a `:qualifier` is new.

With it we can be more confident that we aren't muting untargeted instances of a given warning.

...this :qualifier value so far only is used in the `:suspicious-test`. Other linters could eventually benefit from it, would have to give it a look